modules/distr-tracing-tempo-install-gateway-permissions.adoc

```diff
@@ -0,0 +1,169 @@
+// Module included in the following assemblies:
+//
+// * observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="distr-tracing-tempo-install-gateway-permissions_{context}"]
+= Configuring tenants and permissions
+
+Authentication and authorization are provided by the Tempo Gateway service. OpenShift OAuth and the Kubernetes `TokenReview` API are used for authentication, and the Kubernetes `SubjectAccessReview` API is used for authorization.
+
+To define tenants and manage their read and write access, the distributed tracing stack—built on the Red Hat distribution of OpenTelemetry and Tempo requires a well-configured authorization setup. This setup relies on Kubernetes Role-Based Access Control (RBAC) through ClusterRole and ClusterRoleBinding. By default, no users are granted read or write permissions, ensuring a secure baseline until explicit configurations are defined.
+
+You can install a Configure those permissions from the *Administrator* view of the web console or using command line CLI.
+
+.Prerequisites
+
+* You are logged in to the {product-title} web console as a cluster administrator with the `cluster-admin` role.
+* For {product-dedicated}, you must be logged in using an account with the `dedicated-admin` role.
+
+.Reading traces
+
+To grant users permission to read a specific tenant, follow these steps:
+
+. Define desired tenant name and tenant Id. 
```

Comment:
@rubenvp8510, could you please give me a few lines from the TempoStack CR as an example for these two parameters?

Reply:
I see, no need to. Thank you.